pkg/controllers/configobservation/oauth/observe_tokenconfig_test.go

```diff
@@ -34,6 +34,52 @@ func TestObserveTokenConfig(t *testing.T) {
 			},
 			errors: []error{},
 		},
+		{
+			name: "max age 0 still means default max age",
+			config: &configv1.OAuth{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: "cluster",
+				},
+				Spec: configv1.OAuthSpec{
+					TokenConfig: configv1.TokenConfig{
+						AccessTokenMaxAgeSeconds: 0,
+					},
+				},
+			},
+			previouslyObservedConfig: map[string]interface{}{},
+			expected: map[string]interface{}{
+				"oauthConfig": map[string]interface{}{
+					"tokenConfig": map[string]interface{}{
+						"accessTokenMaxAgeSeconds":    float64(86400),
+						"authorizeTokenMaxAgeSeconds": float64(300),
+					},
+				},
+			},
+			errors: []error{},
+		},
+		{
+			name: "inactivity 0 means disabled",
```

Comment:
I'm confused - how is this different from `"max age 0 still means default max age"`?